nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/utils/ConanFacetUtils.java

```diff
@@ -72,7 +72,7 @@ public static Component findComponent(final StorageTx tx,
         Query.builder()
             .where(P_GROUP).eq(coords.getGroup())
             .and(P_NAME).eq(coords.getProject())
-            .and(P_VERSION).eq(coords.getVersion())
+            .and(P_VERSION).eq(String.format("%s-%s", coords.getVersion(), coords.getChannel()))
             .build(),
```

Comment:
Please create new one method called getComponentVersion in ConanProxyHelper, because this logic use in 2 places

Reply:
Fixed 6493481cc0fc0d14e7d56fcee608ad892add7f2b